meldingen_huidige_jaar.map

```diff
@@ -52,14 +52,15 @@ MAP
   END
 
   LAYER
-    NAME                    "Openbare ruimte en veiligheid, openstaand"
+    NAME                    "veiligheid_openstaand"
     INCLUDE                 "connection/dataservices.inc"
-    DATA                    "geometrie FROM (
-      select * 
+    DATA                    "geometrie_visualisatie FROM (
+      select id, hoofdcategorie, subcategorie, thema, datum_melding, tijdstip_melding, datum_overlast, tijdstip_overlast, externe_status, melding_type, melding_soort, gbd_buurt_naam, gbd_wijk_naam, gbd_ggwgebied_naam, gbd_stadsdeel_naam, bron, laatst_gezien_bron
```

Comment:
I think you also need to `select geometrie_visualisatie` here, and in similar queries.